docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -101,6 +101,14 @@ Values containing `#` or `;` must be quoted using `` ` `` or `"""`.
 - `SSL_MODE`: **disable**: For PostgreSQL only.
 - `PATH`: **data/gitea.db**: For SQLite3 only, the database file path.
 
+## Indexer (`indexer`)
+
+- `ISSUE_INDEXER_PATH`: **indexers/issues.bleve**: Index file used for issue search.
+- `REPO_INDEXER_ENABLED`: **false**: Enables code search (uses a lot of disk space).
+- `REPO_INDEXER_PATH`: **indexers/repos.bleve**: Index file used for code search.
+- `UPDATE_BUFFER_LEN`: **20**: Buffer length of index request.
+- `MAX_FILE_SIZE`: **536870912**: Maximum size in bytes of each index files.
```

Comment:
That's wrong default. Looks like it was forgotten to change it in settings loading code. Should be same as in app.ini.sample `1048576` or `1024 * 1024`. Can you please fix it in this PR?

Reply:
done